web/src/utils/duration.tsx

```diff
@@ -39,3 +39,16 @@ export const formatDuration = (ms: number) => {
   });
   return _.trim(str);
 };
+
+export const getDateFromSecondsString = (seconds: string) => {
```

Comment:
Add return type?  
`export const getDateFromSecondsString = (seconds: string): Date => {`

Reply:
I usually don't specify return types when they are explicit as `return new Date(...)` (maybe a bad habbit) but if we are all aligned on that we can do it everywhere :+1:

Reply:
Leave it as-is.  I don't like code bloat for little gain.  I raised it as a question to get consensus on this.

Reply:
sure :+1: